starter-code/index.html

```diff
@@ -3,40 +3,83 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" type="text/css" href="/Users/briannacho/code/labs/lab-css-spotify-clone/starter-code/css/main.css">
```

Comment:
Don't use this kind of path as it will only work on your computer, use this kind of path :  
"css/main.css"

As it's related to your project folder :)